backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisWebservice.java

```diff
@@ -94,7 +97,8 @@ public static void start(int port) {
         "/:schema",
         (req, res) -> {
           if (!("/" + OIDC_LOGIN_PATH).equals(req.pathInfo())
-              && !("/" + OIDC_CALLBACK_PATH).equals(req.pathInfo())) {
+              && !("/" + OIDC_CALLBACK_PATH).equals(req.pathInfo())
+              && !("/" + ROBOTS_TXT).equals(req.pathInfo())) {
```

Comment:
It this actually necessary? I though sparkjava will fire the rules in create order, so will automatically apply the rule on line 53?

Reply:
```I though sparkjava will fire the rules in create order``` no this is part of the before filter (https://sparkjava.com/documentation#filters), it is necessary, or we should remove the 'redirect with trailing slash' from the before

Reply:
we could also just have the before filter handle the 'dynamic non-schema paths' ( oidc, google-stuff, ..) nad remove the get routes, but thats a bit like re-implementing a new router in the before filter

Reply:
ah, this is because the trailing '/' filter. Lets keep it then like this.

Reply:
yep :(